### ISSUE TITLE
Add Config Menu

### ISSUE THREAD

**Issue by thowan-codes** [thowan-codes/Signals-Symbols#1]
Add a config menu to the native VOTV setting menu.
Should be the final tab of the settings menu and should look similar to other VotV Menus
Tab Title:
- S&S
- Signals and Symbols
- [Satellite Icon] & [Symbols Icon]

BP_ConfigManager - Source of Truth, everything should talk to this to read/write their settings
SG_ConfigSettings - Save data for the Config/settings
S_ConfigData - A data structure for the config data
- ConfigVersion
- S_SymbolConfig
- S_ScreenshotConfig
WBP_ConfigSettings - UI to represent the user choices/loaded defaults.

### Signal Symbol Options
boolean EnableSymbols
enum SymbolType
- Text Labels
- Unique Symbols
- Symbol Count

Custom Customizer:
if Text:
- Freeform fields for L0, L1, L2, and L3
if Symbol:
- Dropdown field with available symbols

boolean CustomColorSymbols
boolean CustomSignalOutlineColor

Color Options:
- Level0SymbolColor
- Level1SymbolColor
- Level2SymbolColor
- Level3SymbolColor
- Level0OutlineColor
- Level1OutlineColor
- Level2OutlineColor
- Level3OutlineColor

### Screenshot Options
Screenshot Key: Print Screen

Screenshot Resolution:
- Native
- Downscaled
Percentage of Downscale: SpinBox: Range from 25-100% with .5 increments

Screenshot Save Location:
Default Location: %LOCALAPPDATA%\VotV\Saved\Screenshots\
Save Location Selector-
Screenshot File Type:
- PNG
- JPEG
